helixbass/line-runner
# Control note duration via MIDI slider

In this PR:
- update hardcoded fractional note duration to be controlled by a currently hardcoded MIDI slider from a currently hardcoded MIDI input

To test:
You'll probably need to change the hardcoded names of the MIDI input and MIDI slider channel/control value in order for it to run + be wired up as expected, but at that point if you adjust that slider as it's running you should be able to adjust the individual note duration across the range of zero - a full sixteenth note
